auth_server/Makefile

```diff
@@ -22,7 +22,7 @@ generate:
 	  github.com/cesanta/docker_auth/auth_server/server/...
 
 build:
-	CGO_ENABLED=0 go build -v --ldflags=--s
+	CGO_ENABLED=0 go build -v --ldflags=--s -pkgdir /tmp/go
```

Comment:
This change is unrelated to this PR, could you raise another PR for this change and revert it here?